src/app/shared/constants/flags/flags-extra.ts

```diff
@@ -22,9 +22,15 @@ export const FLAGS_EXTRA: Flag[] = [
   { bit: 18, name: `NO_SKILLGAIN - creature won't increase weapon skills` },
   { bit: 19, name: `TAUNT_DIMINISH - Taunt is a subject to diminishing returns on this creature` },
   { bit: 20, name: `ALL_DIMINISH - Creature is subject to all diminishing returns as player are` },
-  { bit: 21, name: `KNOCKBACK_IMMUNE - set mostly for dungeon bosses and their summons` },
+  { bit: 21, name: `UNKNOWN` },
   { bit: 22, name: `AVOID_AOE - ignored by aoe attacks (for icc blood prince council npc - Dark Nucleus)` },
   { bit: 23, name: `NO_DODGE - target cannot dodge` },
-  { bit: 24, name: `DUNGEON_BOSS - creature is a dungeon boss (SET DYNAMICALLY, DO NOT ADD IN DB)` },
-  { bit: 25, name: `IGNORE_PATHFINDING - creature ignore pathfinding` },
+  { bit: 24, name: `UNKNOWN` },
+  { bit: 25, name: `UNKNOWN` },
+  { bit: 26, name: `UNKNOWN` },
+  { bit: 27, name: `UNKNOWN` },
+  // { bit: 28, name: `DUNGEON_BOSS - creature is a dungeon boss (SET DYNAMICALLY, DO NOT ADD IN DB)` },
```

Comment:
we can uncomment this

Reply:
Why? It is not to be added in the DB so to minimize mistakes I think it should not even be present